src/main/java/de/tum/in/www1/artemis/web/rest/ProgrammingExerciseParticipationResource.java

```diff
@@ -110,6 +110,23 @@ public ProgrammingExerciseParticipationResource(ProgrammingExerciseParticipation
         // Set exercise back to participation
         participation.get().setExercise(exercise);
 
+        // get the result which belongs to the specific correctionround and save it as the single result in the participation
```

Comment:
I think this comment is misleading, as it is not really being saved is it? Maybe 'set' would be better here

Reply:
you are right, we don't save it to the database here
```suggestion
        // get the result which belongs to the specific correctionround and set it as the single result in the participation
```